app/assets/stylesheets/moderators.scss

```diff
@@ -155,7 +155,39 @@
 }
 
 .mod-container {
-  padding: 30px 0px;
+  padding: 16px 0px;
```

Comment:
We can use some `crayons` variables here.

As long as we call `@import 'config/import';` at the top of the file, we can say this:

`padding: $su-4 0;`

In the `config/_import.scss` file, you'll see documentation for our spacing-units (that's what `su` represents).